### PR TITLE
Add link to the create token docs to README

### PR DESCRIPTION
**Pull request by shomix**
Link to readme: https://github.com/spheronFdn/sdk/blob/add-token-description-to-readme/README.md
image

completes SPH-1432

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ const { uploadId, bucketId, protocolLink, dynamicLinks } = await client.upload(
 );
 ```
 
-- The `SpheronClient` constructor takes an object that has one property `token`.
+- The `SpheronClient` constructor takes an object that has one property `token`. To create the `token`, you can follow the instructions in the [DOCS](https://docs.spheron.network/api/rest-api-references#creating-an-access-token).
```
Comment (rekpero) [inline README.md:66]: It's hard to find this api token section, it's better to create a section after the usage with title `Access Token` and then write down this content in there

**Review by rekpero — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shomix) [reply README.md]: Something like this? https://github.com/spheronFdn/sdk/blob/add-token-description-to-readme/packages/storage/README.md
image

Comment (rekpero) [reply README.md]: yes

Comment (shomix) [reply README.md]: Removed :

**Review by rekpero — DISMISSED** [review]

**Review by dusanstanis — DISMISSED** [review]

**Review by aayushmahapatra — APPROVED** [review]
LGTM

**Review by aayushmahapatra — APPROVED** [review]

**Review by dusanstanis — APPROVED** [review]

**Pull request merged**